data/config-schema/src/main/resources/net/corda/schema/configuration/messaging/1.0/corda.messaging.json

```diff
@@ -99,6 +99,11 @@
       "description": "Settings for publishers that write messages to the underlying bus",
       "$ref": "publisher.json",
       "default": {}
+    },
+    "maxAllowedMessageSize": {
+      "description": "Maxiumum allowed size, in bytes, for publishing Messages. This configuration does _not_ change the Kafka configuration itself.",
```

Comment:
Strictly speaking the underlying bus might not be Kafka, so might be worth rephrasing that this is used by the platform as a guide and has no effect on underlying bus settings.

Reply:
Thanks. Text updated.